submissions/Jemikalajah-Philip/javascript/homepage.js

```diff
@@ -0,0 +1,21 @@
+let imgDiv = document.getElementsByClassName("product");
+let prodTitle = document.getElementsByClassName("title");
+let prodPrice = document.getElementsByClassName("price");
+var productArray = [
+     ["icon/img19.jpg", "Round Neck Polo", "$400"], 
```

Comment:
Uhmm, best to change this to individual objects for readability and future maintenance. 
i.e 
```
[
    {
        name: "V-neck Polo",
        price: "$300",
        image: "icon/img21.jpg",
    },
    .....
]
```
Then access the properties of the object in your loop.